### DISCUSSION TITLE
How to control space of path-to-miner/kvlog/metadata?

### DISCUSSION THREAD

**Discussion by KernelPanic80** [filecoin-project/lotus#8254] [General Q&A] [answered]
Hi!
In one day running lotus-miner kvlog/metadata has 766G
After restart lotus-miner (some time after compacting log) 5,6M

It's related to be so verbose (RUST_LOG="debug")? Or must be controlled with env var (LOTUS_MAX_HEAP)?
What I can do to compact (gc) metadata without restart?

Thanks!

**Comment by KernelPanic80** [comment, marked as answer]
I've found that its related to that issue https://github.com/filecoin-project/lotus/issues/6547#issuecomment-1061181253

**Comment by SgtCoin** [comment]
Marking as answered/archived.
